vendor/bat-native-ads/src/bat/ads/internal/ads/serving/new_tab_page_ad_serving.cc

```diff
@@ -108,12 +108,6 @@ void Serving::ServeAd(const NewTabPageAdInfo& ad,
     return;
   }
 
-  if (ad.wallpapers.empty()) {
```

Comment:
Hmm, this check can help us to avoid the crash in case of wrong data came from the server catalog.